src/rollup/index.ts

```diff
@@ -229,7 +229,7 @@ function checkOutputOptions(options: OutputOptions) {
 
 	if (!options.format) {
```

Comment:
An alternative here might be to do `if (options.format === 'es') options.format = 'esm';` to avoid the need to do both checks through the codebase.

Reply:
I guess this merge should happen in `mergeOptions`?

Reply:
This could work, but there is an issue, what we should pass to transformBundle(code, { format })?

Reply:
It would be a breaking change to alter - so keeping `es` internally and `esm` as an alias is probably the best bet.